sopel/modules/pronouns.py

```diff
@@ -19,14 +19,21 @@
     'ze/hir': 'ze/hir/hir/hirs/hirself',
     'ze/zir': 'ze/zir/zir/zirs/zirself',
     'they': 'they/them/their/theirs/themselves',
+    'they/them': 'they/them/their/theirs/themselves',
     'they/.../themselves': 'they/them/their/theirs/themselves',
     'they/.../themself': 'they/them/their/theirs/themself',
     'she': 'she/her/her/hers/herself',
+    'she/her': 'she/her/her/hers/herself',
     'he': 'he/him/his/his/himself',
+    'he/him': 'he/him/his/his/himself',
     'xey': 'xey/xem/xyr/xyrs/xemself',
+    'xey/xem': 'xey/xem/xyr/xyrs/xemself',
     'sie': 'sie/hir/hir/hirs/hirself',
+    'sie/hir': 'sie/hir/hir/hirs/hirself',
     'it': 'it/it/its/its/itself',
+    'it/it': 'it/it/its/its/itself',
     'ey': 'ey/em/eir/eirs/eirslef',
+    'ey/em': 'ey/em/eir/eirs/eirslef',
```

Comment:
L35 has been wrong for who knows how long. L36 copied the mistake.


```suggestion
    'ey': 'ey/em/eir/eirs/eirself',
    'ey/em': 'ey/em/eir/eirs/eirself',
```